### ISSUE TITLE
Loop becomes unresponsive after some time

### ISSUE THREAD

**Issue by Jossec101** [lightninglabs/loop#767]
After some time (we have not measure how much yet), our loop is having issues doing loop outs, getinfo breaks as in the log below:
```
lnd:/# loop getinfo
[loop] rpc error: code = Unknown desc = sql: Scan error on column index 20, name "publication_deadline": unsupported Scan, storing driver.Value type string into type *time.Time
```

Additionally, invoking loop out gives this error:
```
cltv delta below minimum: 98
```

It somehow resolves if we kill `loopd`  and some swaps got broadcast onchain

**System information**

Versions:
LND 0.17.5 and loop 0.27.1 for amd64 linux containers on kubernetes (AWS)

**Comment by hieblmi** [comment]
Thanks for raising the issue @Jossec101. Have you been running this setup for a while or did this behavior recently just appear or did you upgrade any versions of your stack?

**Comment by Jossec101** [comment]
> Thanks for raising the issue @Jossec101. Have you been running this setup for a while or did this behavior recently just appear or did you upgrade any versions of your stack?

Nope, we have running loopd for more than 1 year upgrading from time to time to newer versions, recently we just started to see this behaviour.

**Comment by Jossec101** [comment]
> Are you using sqlite or PostgreSQL? If PostgreSQL, could you please provide the version that your interfacing with?

SQLite

**Comment by Jossec101** [comment]
Wiping up the db fixed this issue, it looks to be a corruption over time

**Comment by bhandras** [comment]
Thanks for the heads-up @Jossec101. If I understand correctly you're referring to filesystem corruption? Did you identify the source of the corruption? Or do you maybe mean that the DB file itself was changed outside of `loopd`?

**Comment by Jossec101** [comment]
> Thanks for the heads-up @Jossec101. If I understand correctly you're referring to filesystem corruption? Did you identify the source of the corruption? Or do you maybe mean that the DB file itself was changed outside of `loopd`?

I mean that we stopped loopd, moved the sqlite to a safe place and loopd recreated a new one and that solved the issue.

**Comment by Jossec101** [comment]
It happened again today, bump

**Comment by bhandras** [comment]
Did you get the same error as reported in https://github.com/lightninglabs/loop/issues/767#issue-2330500692 ?

**Comment by Jossec101** [comment]
Yes, and and the db actually corrupted after a few hours since last comment, and we had again to recreate it, but we upgraded to loop 0.28, I will report back if it happens again

**Comment by Jossec101** [comment]
It has happened again, actually the frequency is increasing
